docs/design-proposals/2020-09-07-canary-deployments-for-tidb-operator.md

```diff
@@ -0,0 +1,34 @@
+# Canary deployments for TiDB-Operator
+
+## Summary
+
+This document presents a design to support canary deployments TiDB-Operator.
+
+## Motivation
+
+When upgrade TiDB-Operator, users may want to deploy multiple cluster-scoped TiDB-Operators in the same Kubernetes cluster.
+And choose some specific tidb clusters will be managed by new  version TiDB-Operator, the others are still managed by old version. When the new version TiDB-Operator is fully tested, replace the old one.
+
+### Goals
+
+* Deploy multiple TiDB-Operator in one Kubernetes cluster
+* Manage tidbcluster which be selected
+* Don't manage `tidbcluster` which be excluded
+
+### Non-Goals
+
+* Upgrading `TiDB-Operator`
+* Upgrading `tidbcluster`
+
+## Proposal
```

Comment:
Could you please add more detail?
I think there are two issues here:

- TiDB Operator supports managing TidbClusters by label selector, which we can add configuration in the values.yaml of the TiDB Operator chart and pass it as an option [here](https://github.com/pingcap/tidb-operator/blob/master/cmd/controller-manager/main.go#L162)
- Support to deploy multiple cluster-scoped TiDB Operators in the same Kubernetes cluster. Can this be done easily with Helm?